src/middlewared/middlewared/plugins/cron.py

```diff
@@ -41,14 +48,38 @@ def cron_extend(self, data):
                     'Specified user does not exist'
                 )
 
+        command = data.get('command')
+        if not command:
+            verrors.add(
+                f'{schema}.command',
+                'Please specify a command for cronjob task.'
+            )
+        else:
+            allowed_length = 1000 - len(' '.join(
```

Comment:
Ah, also a comment about why the magical 1000, please. The next developer working here will appreciate ;)